Cookbook/Technical-Documents/PullAssigners.md

```diff
@@ -0,0 +1,104 @@
+# How to configure Pull Assigners for Babylon repos
+
+## Introduction
+
+[Pull Panda](https://pullpanda.com/) is a set of tools for automating actions on GitHub Pull Requests and provide some integration with Slack. Amongst those tools, Pull Assigner will allow you to auto-assign people from your team as reviewers of Pull Requests, typically trying to distribute the PRs amongst the team evenly
+
+The aim of this document is to explain how to configure `Pull Assigner` on our Babylon repos in order for it to affect people to PRs automatically
+
+- We will use the `CODEOWNERS` file to make GitHub always assign a special GitHub team (called the proxy team in Pull Assigner's parlance) to all our Pull Requests
+- That special GitHub team will in fact be managed by Pull Assigners, which will detect when that team is added as a reviewer
+- Pull Assigner will then pick N reviewers from a GitHub team listing all possible reviewers, then unassign the "proxy team"
+- Optionally, we could also require each PR of a repository to always have an approval from a subset of people (the ones owning the responsibility of the code) – like we did for our bots repository owned by the Developer Experience squad
+
+## Configure GitHub Teams
+
+### A team to list the reviewers to pick from
+
+1. First you need a GitHub team containing the list of all the people you want Pull Assigner to pick reviewers from.
+
+   * For the iOS repos, we use the team `@Babylonpartners/iOS-Admin` for this, which should contain every iOS developer that are on our team. So if you need the same list of reviewers, you can just use this one instead of creating one.
+   * If you need a different list to pick your reviewers from, you'd need to create a team via [this page on GitHub](https://github.com/orgs/Babylonpartners/new-team), and once it's created, go to the team's page and add Members to that new team ([e.g. here for `ios-admin`](https://github.com/orgs/Babylonpartners/teams/ios-admin/members))
```

Comment:
what do we use `ios-admin` for? it's confusing to have two teams with just different case in the name, we already know that from our mailing lists 🙄

Reply:
Oh, `iOS-Admin` and `ios-admin` are the same, it's just that GitHub team names are case-insnsitive. I'll amend the doc to use `iOS-Admin` casing everywhere.

```suggestion
   * If you need a different list to pick your reviewers from, you'd need to create a team via [this page on GitHub](https://github.com/orgs/Babylonpartners/new-team), and once it's created, go to the team's page and add Members to that new team ([e.g. here for `iOS-Admin`](https://github.com/orgs/Babylonpartners/teams/iOS-Admin/members))
```